src/utils/payloadStringify.test.js

```diff
@@ -108,6 +114,47 @@ describe('payloadStringify', () => {
     expect(result).toEqual('{"a":2,"password":"****"}');
   });
 
+  test('payloadStringify -> should not mask masking envs', () => {
+    const someValue = 'a';
+
+    const envs = [
+      LUMIGO_SECRET_MASKING_REGEX,
+      LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP,
+      LUMIGO_SECRET_MASKING_REGEX_HTTP_REQUEST_BODIES,
+      LUMIGO_SECRET_MASKING_REGEX_HTTP_REQUEST_HEADERS,
+      LUMIGO_SECRET_MASKING_REGEX_HTTP_RESPONSE_BODIES,
+      LUMIGO_SECRET_MASKING_REGEX_HTTP_RESPONSE_HEADERS,
+      LUMIGO_SECRET_MASKING_REGEX_ENVIRONMENT,
+      LUMIGO_SECRET_MASKING_REGEX_HTTP_QUERY_PARAMS,
+      LUMIGO_SECRET_MASKING_EXACT_PATH,
+      LUMIGO_SECRET_MASKING_DEBUG,
+    ];
+
+    envs.forEach((env) => {
+      expect(
+        payloadStringify({
+          [env]: someValue,
+        })
+      ).toEqual(`{"${env}":"${someValue}"}`);
+    });
+  });
```

Comment:
This can be shortened to the more conventional `test.each` structure:
```suggestion 
  test.each([
      LUMIGO_SECRET_MASKING_REGEX,
      LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP,
      LUMIGO_SECRET_MASKING_REGEX_HTTP_REQUEST_BODIES,
      LUMIGO_SECRET_MASKING_REGEX_HTTP_REQUEST_HEADERS,
      LUMIGO_SECRET_MASKING_REGEX_HTTP_RESPONSE_BODIES,
      LUMIGO_SECRET_MASKING_REGEX_HTTP_RESPONSE_HEADERS,
      LUMIGO_SECRET_MASKING_REGEX_ENVIRONMENT,
      LUMIGO_SECRET_MASKING_REGEX_HTTP_QUERY_PARAMS,
      LUMIGO_SECRET_MASKING_EXACT_PATH,
      LUMIGO_SECRET_MASKING_DEBUG
    ], 'payloadStringify -> should not mask masking the env-var %s', (envVar) => {
      const someValue = 'a';
      
      expect(payloadStringify({ [envVar]: someValue })).toEqual(`{"${env}":"${someValue}"}`);
  });
```